ynput/ayon-unreal
# Fix the incorrect sequence path when using pre-hierarchy for render creator

## Changelog Description
When we create render instance with pre-hierarchy options enabled, it will raise the error and the system can't manage to find the correct path. This PR is to fix it.


## Additional info
We should allow user settings for the sequences path
Resolve #58 

## Testing notes:

1. Create some level sequences inside `/Game/Ayon/Sequences/`
2. Create Render Instance with pre-hierarchy
3. Should not be error out

Also testing the steps below: 
1. Produce any rendered frames via ```AYON>Render``` (by render instance)
2. Use "Publish" with that render
3. Publishing fails on integration part
